src/cronselection.html

```diff
@@ -7,71 +7,74 @@
+                <option ng-repeat="value in monthValues" ng-selected="myFrequency.monthValues.indexOf(value) >= 0" value="{{value}}">
                     {{value | cronMonthName}}
                 </option>
             </select>
         </div>
         <span ng-show="myFrequency.base >= 2">at </span>
+        <!-- If Multiple is Enabled -->
         <div ng-show="myFrequency.base >= 3" class="cron-select-wrap">
-            <select class="cron-select hour-value" ng-model="myFrequency.hourValue" ng-if="allowMultiple" multiple>
-                <option ng-repeat="value in hourValue" ng-selected="myFrequency.hourValue.indexOf(value) >= 0" value="{{value}}">
+            <select class="cron-select hour-value" ng-model="myFrequency.hourValues" ng-if="allowMultiple" multiple>
+                <option ng-repeat="value in hourValues" ng-selected="myFrequency.hourValues.indexOf(value) >= 0" value="{{value}}">
                     {{value}}
                 </option>
             </select>
-
-            <select class="cron-select hour-value" ng-model="myFrequency.hourValue" ng-if="!allowMultiple">
-                <option ng-repeat="value in hourValue" ng-selected="myFrequency.hourValue.indexOf(value) >= 0" value="{{value}}">
+            <!-- If Multiple is not Enabled -->
+            <select class="cron-select hour-value" ng-model="myFrequency.hourValues" ng-if="!allowMultiple">
+                <option ng-repeat="value in hourValues" ng-selected="myFrequency.hourValues.indexOf(value) >= 0" value="{{value}}">
                     {{value}}
                 </option>
             </select>
         </div>
         <span ng-show="myFrequency.base >= 3"> : </span>
+        <!-- If Multiple is Enabled -->
         <div ng-show="myFrequency.base >= 2" class="cron-select-wrap">
-            <select class="cron-select minute-value" ng-model="myFrequency.minuteValue"  ng-if="allowMultiple" multiple>
-                <option ng-repeat="value in minuteValue" ng-selected="myFrequency.minuteValue.indexOf(value) >= 0" value="{{value}}">
+            <select class="cron-select minute-value" ng-model="myFrequency.minuteValues" ng-if="allowMultiple" multiple>
+                <option ng-repeat="value in minuteValues" ng-selected="myFrequency.minuteValues.indexOf(value) >= 0" value="{{value}}">
                     {{value}}
                 </option>
             </select>
-
-            <select class="cron-select minute-value" ng-model="myFrequency.minuteValue"  ng-if="!allowMultiple">
-                <option ng-repeat="value in minuteValue" ng-selected="myFrequency.minuteValue.indexOf(value) >= 0" value="{{value}}">
+            <!-- If Multiple is not Enabled -->
+            <select class="cron-select minute-value" ng-model="myFrequency.minuteValues" ng-if="!allowMultiple">
+                <option ng-repeat="value in minuteValues" ng-selected="myFrequency.minuteValues.indexOf(value) >= 0" value="{{value}}">
```

Comment:
The thing is, I've tried doing exactly that at first in my original PR. Even though the `selected` attribute appears, the options does not appear selected in the browser itself. And the only way to see that is by looking at the browser. I'm not sure how to capture this via tests.
Anyway, the use of `ng-options` (in my original PR) solves this issue.